daprdocs/content/en/reference/components-reference/supported-configuration-stores/azure-appconfig-configuration-store.md

```diff
@@ -0,0 +1,91 @@
+---
+type: docs
+title: "Azure App Configuration"
+linkTitle: "Azure App Configuration"
+description: Detailed information on the Azure App Configuration configuration store component
+aliases:
+  - "/operations/components/setup-configuration-store/supported-configuration-stores/setup-azure-appconfig/"
+---
+
+## Component format
+
+To set up an Azure App Configuration configuration store, create a component of type `configuration.azure.appconfig`.
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+spec:
+  type: configuration.azure.appconfig
+  version: v1
+  metadata:
+  - name: host
+    value: <HOST>
+  - name: connectionString
+    value: <CONNECTIONSTRING>
+  - name: maxRetries
+    value: # Optional
+  - name: retryDelay
+    value: # Optional
+  - name: maxRetryDelay
+    value: # Optional
+  - name: azureEnvironment # Optional, defaults to AZUREPUBLICCLOUD
+    value: "AZUREPUBLICCLOUD"
+  # See authentication section below for all options
+  - name: azureTenantId # Optional
+    value: "[your_service_principal_tenant_id]"
+  - name: azureClientId # Optional
+    value: "[your_service_principal_app_id]"
+  - name: azureCertificateFile # Optional
+    value : "[pfx_certificate_file_fully_qualified_local_path]"
+
+```
+
+{{% alert title="Warning" color="warning" %}}
+The above example uses secrets as plain strings. It is recommended to use a secret store for the secrets as described [here]({{< ref component-secrets.md >}}).
+{{% /alert %}}
+
+## Spec metadata fields
+
+| Field             | Required | Details | Example |
+|-------------------|:--------:|---------|---------|
+| host              | N        | Endpoint for the Azure App Configuration instance. No Default  | `https://dapr.azconfig.io`
```

Comment:
@cmendible Quick question, mention that host will be used with the Azure AD based auth? 
And also make it clear that Azure AD needs host for sure. 
See https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-azure-eventhubs/ for more details on the either/or required fields. 
In comments in the component format add in details regarding the when host is used with which Auth and when connection string is used.